_data/monthly-calls.yml

```diff
@@ -0,0 +1,104 @@
+    subtitle: Developing and evaluating content for manual accessibility and user experience (UX) tests
+    description: |
+      We talked about the easy-to-follow, manual accessibility and user experience (UX) tests we're developing for USWDS components.
+      We're calling these tests Critical Checklists.
+    date: June 2023
+    id: UxC-a48Vn7w?si=sJU-EAZmJfVRMEc-
+    event_link: https://digital.gov/event/2023/06/15/uswds-monthly-call-june-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-june-2023.pptx
+      size: 2.0
+      pages: 79
+  - title: Real-world accessibility
+    subtitle: What's it like to be blind and use the web in 2023
+    description: To observe Global Accessibility Awareness Day we talked about the real-world experience of blind users on the web.
+    date: May 2023
+    id: SG6Bbv-uKeg?si=9nzftzWV-V9MEtzQ
+    event_link: https://digital.gov/event/2023/05/18/uswds-monthly-call-may-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-may-2023.pptx
+      size: 5.4
+      pages: 23
+  - title: Top tasks
+    subtitle: What the U.S. Web Design System team did and what they found
+    description: |
+      We discussed how and why the USWDS team performed their Top Tasks research,
+      as well as what they learned and what they'll do next.
+    date: April 2023
+    id: d1wPuISaHIg?si=Zx-IcO31nqL4m0C9
+    event_link: https://digital.gov/event/2023/04/20/uswds-monthly-call-april-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-april-2023.pptx
+      size: 3.2
+      pages: 72
+  - title: "Making sense: Attention and the brain"
+    subtitle: Cognition, perception, and design
+    description: |
+      The design system team took a step back and explored how our design decisions both support
+      and are supported by the ways our bodies and brains perceive order and make sense of the world.
+    date: March 2023
+    id: ceRXbLwEgD0?si=13Kf0KTYC7Z0HjSQ
+    event_link: https://digital.gov/event/2023/03/28/uswds-monthly-call-march-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-march-2023.pptx
+      size: 1.8
+      pages: 56
+  - title: Using design tokens
+    subtitle: What design tokens are and how to adapt them to agency needs
+    description: |
+      Take a deep dive into USWDS design tokens. What are they? How are they used?
+      How can they be adapted to agencies' branding needs?
+    date: February 2023
+    id: ObqPeagLZ5o?si=bmjxNX6e660J8CyR
+    event_link: https://digital.gov/event/2023/02/16/uswds-monthly-call-february-2023/
+    slides:
+      link: https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-february-2023.pptx
+      size: 14.6
+      pages: 67
+  - title: Frequently asked questions
+    subtitle: Answering your most-asked questions about the U.S. Web Design System
+    date: January 2023
```

Comment:
@amyleadem - adding a description for this one, too: The USWDS team answers frequent questions, such as how to theme USWDS, how to set up custom fonts, and when new components get released.

Reply:
Added in [205ba7a](https://github.com/uswds/uswds-site/pull/2345/commits/205ba7a89b02e09ae5b397a1f71c786faa54fa84).